src/main/java/com/box/sdk/BoxAPIRequest.java

```diff
@@ -371,9 +371,9 @@ public BoxAPIResponse send() {
      */
     public BoxAPIResponse send(ProgressListener listener) {
         if (this.api == null) {
-            this.backoffCounter.reset(BoxGlobalSettings.getMaxRequestAttempts() + 1);
+            this.backoffCounter.reset(BoxGlobalSettings.getMaxRetryAttempts() + 1);
         } else {
-            this.backoffCounter.reset(this.api.getMaxRequestAttempts() + 1);
+            this.backoffCounter.reset(this.api.getMaxRetryAttempts() + 1);
```

Comment:
Adding + 1 is the way to go, other wise we'd have to modify BackoffCounter.decrement() which seems too far.